### DISCUSSION TITLE
Can't login after suspend / lock

### DISCUSSION THREAD

**Discussion by MrDerekib** [mylinuxforwork/dotfiles#367] [General]
If a go AFK for a while, the pc promts a login screen (black lock screen with a circle with an image) I can't go to desktop again... it says my password is not valid (I've tried user and root password).

I've replicated it several times and always have to reboot my pc to login again.

It's there any config file or something I have to look at?

**Comment by ja-softdevel** [comment]
I'm seeing the same issue. If laptop locks or suspends, I get a blurred background, circle with an image, but no response to key presses or mouse clicks.

**Comment by mylinuxforwork** [comment]
@MrDerekib @ja-softdevel Here is a helpful article https://github.com/hyprwm/hyprlock/issues/101

Can you try the solution mentioned at the bottom, reboot and try again?

Thanks in advance.
